docs/add-ons/configuration.md

```diff
@@ -112,6 +112,9 @@ ports:
 map:
   - config:rw
   - ssl
+  - type: config
```

Comment:
Let's only document the new style and let's face out the old format

Reply:
Sounds great! I updated the documentation to use the new format.